### PR TITLE
Added more Keywords to emoji.json

### PR DESCRIPTION
**Pull request by Sowmi3thri**
Hi @muan ,
Have added some keywords to emoji.json.
Please consider them for merging.
Thanks

### CODE REVIEW

File: emojis.json [hunk 1]
```diff
@@ -194,19 +194,19 @@
   "speech_balloon": ["bubble", "words", "message", "talk", "chatting"],
   "footprints": ["feet", "tracking", "walking"],
   "thought_balloon": ["bubble", "cloud", "speech", "thinking"],
-  "dog": ["animal", "friend", "nature", "woof", "puppy"],
+  "dog": ["animal", "friend", "nature", "woof", "puppy", "pet", "faithful"],
   "wolf": ["animal", "nature", "wild"],
-  "cat": ["animal", "meow", "nature"],
+  "cat": ["animal", "meow", "nature", "meow", "pet"],
```
Comment (muan) [inline emojis.json]: `meow` is already a keyword

File: emojis.json [hunk 2]
```diff
@@ -194,19 +194,19 @@
...
   "mouse": ["animal", "nature", "cheese"],
   "hamster": ["animal", "nature"],
-  "rabbit": ["animal", "nature"],
-  "frog": ["animal", "nature"],
-  "tiger": ["animal", "cat", "danger", "wild", "nature"],
+  "rabbit": ["animal", "nature", "pet", "carrot"],
```
Comment (muan) [inline emojis.json]: I don't think `rabbit` is an expected result for `carrot`.

File: emojis.json [hunk 3]
```diff
@@ -235,28 +235,28 @@
...
   "whale2": ["animal", "nature", "sea", "ocean"],
-  "cow2": ["beef", "ox", "animal", "nature"],
+  "cow2": ["beef", "ox", "animal", "nature", "moo", "milk"],
   "ram": ["animal", "sheep", "nature"],
   "rat": ["animal", "mouse", "rodent"],
   "water_buffalo": ["animal", "nature", "ox", "cow"],
-  "tiger2": ["animal", "nature"],
-  "rabbit2": ["animal", "nature"],
+  "tiger2": ["animal", "nature", "roar"],
+  "rabbit2": ["animal", "nature", "pet", "carrot"],
```
Comment (muan) [inline emojis.json]: same as above

File: emojis.json [hunk 4]
```diff
@@ -544,7 +544,7 @@
   "tm": ["trademark", "brand"],
   "beginner": ["badge", "shield"],
   "kr": ["korea", "nation", "flag", "country", "banner"],
-  "toilet": ["restroom", "wc"],
+  "toilet": ["restroom", "wc", "washroom", "wash"],
```
Comment (muan) [inline emojis.json]: `washroom` has `wash`

**Comment by Sowmi3thri** [comment]
Updated the file considering your comments.

**Comment by muan** [comment]
Thanks :sparkles:

**Pull request merged**